pd/src/state.rs

```diff
@@ -595,4 +595,23 @@ ON CONFLICT (id) DO UPDATE SET data = $1
             })
             .collect())
     }
+
+    /// Retrieve the delegation changes for the supplied epoch
+    /// TODO: should we have a DelegationChanges struct instead of just returning a BTreeMap?
```

Comment:
Possibly -- do we need it to do anything special or maintain any invariants, or would a type alias work?